blueprints/app/files/config/ember-cli-update.json

```diff
@@ -0,0 +1,18 @@
+{
+  "schemaVersion": "1.0.0",
+  "packages": [
+    {
+      "name": "ember-cli",
+      "version": "<%= emberCLIVersion %>",
+      "blueprints": [
+        {
+          "name": "<%= blueprint %>",
+          "outputRepo": "https://github.com/ember-cli/ember-<%= blueprint === 'app' ? 'new' : 'addon' %>-output",
```

Comment:
I don't see `outputRepo` in the associated RFC, and I only found minimal documentation on `ember-cli-update`'s guides -- will the `outputRepo` generated be valid when generating with options that that don't match the ones that we use to generate those repos? For example, if I generate with `--yarn` and this file has both `ember-new-output` as the `outputRepo` and `--yarn` in the options, what happens?

Reply:
The options (and version) specified in the file are used to generate (from scratch) the source diff. So specifically, using this system will actually mean you get correct diffs applied when in a project that uses yarn.

Reply:
Great -- that wasn't clear to me from the documentation on ember-cli-update. It sounded like `outputRepo` was a thing to specify only if it could be used in place of the local generation & diffing.

Reply:
Katie you are right. outputRepo is the old style, where you don't generate the diff locally. It is still used if you run `ember-cli-update compare`, to get a quick diff on github, without doing a bunch of work locally. It can also be used by custom blueprints to keep track of their diffs in the same way.

Reply:
@kellyselden so should I remove it? Also why does `npx ember-cli-update bootstrap` emit it?

Reply:
I would say keep it. It's a super quick way to see what has changed. `ember-cli-update compare` opens it up in your browser with the diff.

Reply:
kk, thanks again @kellyselden!